gips/data/landsat/landsat.py

```diff
@@ -229,7 +225,7 @@ class landsatAsset(gips.data.core.CloudCoverAsset,
         # DN & SR assets are no longer fetchable
         'DN': {
             'sensors': ['LT5', 'LE7', 'LC8'],
-            'enddate': date(2017, 4, 30),
+            'enddate': date.today(),
```

Comment:
Sorry I accidentally resolved the comment that was here. I do not see a need to keep this today()